README.md

```diff
@@ -23,55 +23,57 @@ define a common infrastructure for the bootloader, system flash layout on
 microcontroller systems, and to provide a secure bootloader that enables
 simple software upgrades.
 
-MCUboot is operating system and hardware independent and relies on
-hardware porting layers from the operating. Currently, MCUboot works
-with both the Apache Mynewt and Zephyr operating systems, but more
-ports are planned in the future. RIOT is currently supported as a boot
-target with a complete port planned.
+MCUboot is operating-system and hardware independent, and relies on
+hardware porting layers from the operating system it works with. Currently,
+MCUboot works with the following operating systems:
+- [Zephyr](https://www.zephyrproject.org/)
+- [Apache Mynewt](https://mynewt.apache.org/)
+- [Apache NuttX](https://nuttx.apache.org/)
+- [RIOT](https://www.riot-os.org/)
+- [Mbed OS](https://os.mbed.com/)
+- [Espressif IDF](https://idf.espressif.com/)
```

Comment:
We should probably also include the Cypress/Infineon port. https://www.cypress.com/ is their site, their docs are in boot/cypress/README.md

Reply:
fixed